bigbluebutton-config/bin/bbb-conf

```diff
@@ -1179,7 +1185,7 @@ check_state() {
         echo "#"
     fi
 
-    if [ "$(yq r /usr/share/meteor/bundle/programs/server/assets/app/config/settings.yml public.media.sipjsHackViaWs)" != "true" ]; then
+    if [ "$(echo "$HTML5_CONFIG" | yq r - public.media.sipjsHackViaWs)" == "false" ]; then
```

Comment:
In this case likely != true was handling also undefined or null values. Do we handle them now via yq in the merge of `HTML5_CONFIG`?